mantle/kola/tests/ignition/qemufailure.go

```diff
@@ -101,3 +114,69 @@ func ignitionFailure(c cluster.TestCluster) error {
 		return nil
 	}
 }
+
+func bootfsFailure(c cluster.TestCluster) error {
+	// We can't create files in / due to the immutable bit OSTree creates, so
+	// this is a convenient way to test Ignition failure.
+	failConfig, err := conf.EmptyIgnition().Render(conf.FailWarnings)
+	if err != nil {
+		return errors.Wrapf(err, "creating empty config")
+	}
+	cmd := exec.Command("/bin/bash", "-c", fmt.Sprintf(`set -euo pipefail;
+			truncate -s 1G fakeboot
+			mkfs.ext4 -L boot fakeboot
+		 `))
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	if err := cmd.Run(); err != nil {
+		c.Fatal(err)
+	}
+	// failConfig.AddFile("/notwritable.txt", "Hello world", 0644)
+
+	builder := platform.NewQemuBuilder()
+	defer builder.Close()
+	builder.SetConfig(failConfig)
+	err = builder.AddBootDisk(&platform.Disk{
+		BackingFile: kola.QEMUOptions.DiskImage,
+	})
```

Comment:
We need another `builder.AddDisk()` around here to add the fakeboot disk.

Reply:
Here's where I have a question. 
```
err = builder.AddBootDisk(&platform.Disk{
	BackingFile: kola.QEMUOptions.DiskImage,
```
produces (according to qemu.go](https://github.com/coreos/coreos-assembler/blob/main/mantle/platform/qemu.go#L90) raw disk image.
I'm not sure at this stage, how to call `builder.AddDisk()` correctly, to reference the `fakeboot` disk.

Reply:
@jlebon - please see the [update](https://github.com/coreos/coreos-assembler/blob/0bc49aad7def77fa5228d4cb87f8c26aae71d24b/mantle/kola/tests/ignition/qemufailure.go#L147-L160) and the [builder](https://github.com/coreos/coreos-assembler/blob/0bc49aad7def77fa5228d4cb87f8c26aae71d24b/mantle/kola/tests/ignition/qemufailure.go#L182-L198)

Reply:
You can set the `BackingFile` field to the path to the file you've created and `BackingFormat` to `raw`.

I think that'll make the builder create a qcow2 overlay for it which is unnecessary in our case but meh, it's harmless.